src/System.Security.Cryptography.Cng/tests/CreateTests.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public static class CreateTests
+    {
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public static void CreateEmphemeral1()
```

Comment:
Rather than just "1" and "2", can the test names describe what they're doing (in particular, different from each other?)